### DISCUSSION TITLE
git add . issue

### DISCUSSION THREAD

**Discussion by LTada1** [community/community#40782] [New to GitHub]
Please, how can i tackle this error message when i try to git add . : fatal: Unable to create '/root/alx-system_engineering-devops/.git/index.lock': Disk quota exceeded
?
please someone should help me

**Comment by mohamedzhioua** [comment]
Try :
rm -f ./.git/index.lock

In your repository directory. The error message is rather explicit as to what causes it typically, so if you have no other git processes running (which is the normal case), go ahead and delete that file.

**Comment by airtower-luna** [comment]
> Disk quota exceeded

The error message means there's a limit to how much data (MB, GB, …) you are allowed to use, and it's filled up. Either delete stuff you don't need any more, or talk to an admin and ask for a larger quota.

Also you probably shouldn't be working as root, though the disk quota implies you're in a container or similar, which limits the possible damage. :sweat_smile:

**Comment by mohamedzhioua** [comment]
@LTada1 you can take a look here in the below link : "https://nitayneeman.com/posts/how-to-solve-index-file-exists-error-in-git-submodules/"
